politeiawww/cmsuser.go

```diff
@@ -465,3 +466,91 @@ func convertCMSUserFromDatabaseUser(user *user.CMSUser) cms.User {
 		SupervisorUserID:                user.SupervisorUserID,
 	}
 }
+
+// issuanceDCCUser does the processing to move a nominated user to a fully
+// approved and invite them onto CMS.
+func (p *politeiawww) issuanceDCCUser(userid, sponsorUserID string, domain, contractorType int) error {
```

Comment:
Should this be returning user errors?  User errors should only be returned when the user does something wrong.  Same with `RevokeDCCUser`.